static/admin/config.yml

```diff
@@ -0,0 +1,26 @@
+backend:
+  name: "github"
+  #repo: "OpenRailAssociation/osrd-website"
+  repo: "emersion/osrd-website"
+site_url: "https://osrd.fr"
+media_folder: "static/images"
+public_folder: "/images"
+#publish_mode: editorial_workflow
```

Comment:
With the default publish mode, Decap CMS pushes directly to the master branch. With the editorial workflow, Decap CMS pushes to a dedicated branch and creates a PR. This will probably be what we want.